config/locales/es.yml

```diff
@@ -135,4 +153,17 @@ es:
               blank: 'La posicion secundaria no debe estar en blanco'
             strong_foot: 
               blank: 'Tú pie fuerte no debe estar en blanco'
```

Comment:
Oh, I didn't see this typo on the first CR of this part, could you remove the acento please?